BetterJoyForCemu/Config.cs

```diff
@@ -57,7 +57,7 @@ public static class Config { // stores dynamic configuration, including
 				}
 
 				// if old settings
-				if (lineNO < settingsNum) {
+				if (lineNO < settingsNum + 1) {
```

Comment:
Ah, I found a bug where the settings file would not be reset because lineNO is 1 based and settingsNum is 0 based.
I could change the lineNO at line 29 to start at -1 that should also fix it.

Reply:
Changed

Reply:
As it turns out changing lineNO to -1 causes it to not read the settings file correctly.

Reply:
Commit 238a192 fixes it